### PR TITLE
 Slight refactor of readme, and additional information, for clarity

### PR DESCRIPTION
**Pull request by josh-fat-potato-uk**
Made minikube the primary way to start using Kube over Docker-desktop, due to auth issues
Added clarity on a few things
Added info on how to port-forward a service

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,26 +2,13 @@
...
-*Note this will take a few minutes depending on your connection*
-
-We can then switch to this cluster using the Docker desktop icon in the tool bar at the top
-
-![docker desktop toolbar](docker-desktop-switch-context.png)
-
-
-<details>
-<summary><i> minikube alternative</i></summary>
 If Docker desktop is not an option, we can also use something called <code>minikube</code>
```
Comment (Jack-fat-potato-uk) [inline README.md]: Might be worth moving this line down to the Docker desktop alternative section?

File: README.md [hunk 2]
```diff
@@ -280,6 +283,11 @@ We will now be able to see on the dashboard that our service is of type `NodePor
 kubectl get service sample-app-service --output='jsonpath="{.spec.ports[0].nodePort}"'
 ```
 
+You also need to forward the port for the service. This differs slightly from forwarding a pod, as you can see below.
```
Comment (Jack-fat-potato-uk) [inline README.md:284]: This is actually very useful, didnt know you could port forward a service 🙂

**Review by Jack-fat-potato-uk — APPROVED** [review]
Nice update, sorry if I have marked the review wrong. I usually avoid github 😂

File: README.md (hunk 1 above)
Comment (josh-fat-potato-uk) [reply README.md]: I've rewritten this a bit and moved it to the alt section

File: README.md (hunk 2 above)
Comment (josh-fat-potato-uk) [reply README.md:284]: Yep! That's what I assumed was meant in the section heading xD Took me a while to find the exact syntax, but it worked well.

**Pull request merged**